pages/common/man.md

```diff
@@ -2,6 +2,10 @@
 
 > Format and display manual pages.
 
+- Display manual page for man:
+
+`man man`
```

Comment:
This is already covered in the next example. I'm not sure in the value of this ?

Reply:
I agree that this is already covered. :+1: